examples/s3/README.md

```diff
@@ -14,10 +14,10 @@ module "s3" {
   source = "coralogix/aws/coralogix//modules/s3"
 
   coralogix_region   = "Europe"
-  CustomDomain       = "https://<your custom doamin>/api/v1/logs"
+  custom_domain       = "https://<your custom doamin>/api/v1/logs"
```

Comment:
indentation

Reply:
If you have coralogix_region configured no need for custom_domain and vise-versa